src/engine/loopingcontrol.cpp

```diff
@@ -546,11 +631,29 @@ void LoopingControl::slotReloopExit(double val) {
         if (loopSamples.start != kNoTrigger && loopSamples.end != kNoTrigger &&
                 loopSamples.start <= loopSamples.end) {
             setLoopingEnabled(true);
+            // If we're not playing, jump to the loop in point so the waveform
+            // shows where it will play from when playback resumes.
+            if (!m_pPlayButton->toBool()) {
+                slotLoopInGoto(1);
+            }
         }
         //qDebug() << "reloop_exit looping on";
     }
 }
 
+void LoopingControl::slotReloopCue(double pressed) {
```

Comment:
It took me quite a while to figure out what it does. 
How about call this control "loop_in_gotoandstop" similar to "cue_gotoandstop"